tox.ini

```diff
@@ -1,19 +1,18 @@
 [tox]
 requires = tox-venv
 skipsdist = True
-envlist = py36,py37,py38,py39,py38-check-format
+envlist = py36,py37,py38,py39
 
 [testenv]
 deps = .[dev]
 commands =
+        black --check tests_selector --exclude helper_project
```

Comment:
Different versions of Python might (theoretically) have different versions of black; also there's no need to check formatting 4 times. That's why I had it in a separate testenv.

Reply:
it is clear but, I wasn't sure how to configure it with github actions, can you give me a hand her?

Reply:
Just omit black call from `[testenv]` and add the following in the end of the `tox.ini`:
```
# No need to run black with every possible Python version
[testenv:py38]
commands_pre = black --check tests_selector --exclude helper_project
```

Now for py38 environment black will be run before anything else.

Reply:
done